roles/edpm_podman/tasks/install.yml

```diff
@@ -107,8 +107,8 @@
       when: not edpm_podman_disconnected_ocp
 
     - name: Write containers registries.conf
-      ansible.builtin.template:
-        src: "{{ edpm_podman_registries_conf }}"
+      ansible.builtin.copy:
+        content: "{{ edpm_podman_registries_conf }}"
```

Comment:
this LGTM as a fix.
I missed reviewing the original patch that added the variable. I think we need a way to distinguish between variables that are meant to be set by our code (openstack-operator in this case), and user variables. As it is now, a user has no way of knowing that they shouldn't use this variable themselves, and if they do, it might not even take effect as they intend.

Just looking at the variable, and the description in argument_specs.yml, as I user I would expect I could use this to define the registries.conf content. When in fact, it only works with disconnected, and could also be overwritten by openstack-operator.

I think we should file an issue to clean this sort of thing up across all the code, b/c I'm sure there are other examples.